internal/server/instance/drivers/qmp/commands.go

```diff
@@ -74,6 +74,35 @@ type CPUModel struct {
 	Flags map[string]any `json:"props"`
 }
 
+// MemoryDevice contains information about a memory device.
+type MemoryDevice struct {
+	Type string          `json:"type"`
+	Data json.RawMessage `json:"data"`
+}
+
+// PcDimmDevice contains information about a memory device of type pc-dimm.
+type PcDimmDevice struct {
```

Comment:
Nit, should be capitalized as `PCIDimmDevice` I think.

Reply:
Done.